libkbfs/interfaces.go

```diff
@@ -1518,11 +1515,15 @@ type BareRootMetadata interface {
 	GetTLFCryptKeyParams(keyGen KeyGen, user keybase1.UID, key CryptPublicKey) (
 		TLFEphemeralPublicKey, EncryptedTLFCryptKeyClientHalf,
 		TLFCryptKeyServerHalfID, bool, error)
-	// IsValidAndSigned verifies the BareRootMetadata given the current
-	// user and device (identified by the KID of the device verifying
-	// key), checks the writer signature, and returns an error if a
-	// problem was found.
-	IsValidAndSigned(codec Codec, crypto cryptoPure,
+	// IsValidAndSigned verifies the BareRootMetadata, checks the
+	// writer signature, and returns an error if a problem was
+	// found.
+	IsValidAndSigned(codec Codec, crypto cryptoPure) error
+	// IsLastModifiedBy verifies that the BareRootMetadata is written by
+	// the given user and device (identified by the KID of the device
+	// verifying key), and returns an error if not. Should be called only
```

Comment:
Or I guess maybe it'll panic if called with an invalid/unsigned md?  If that's the case, maybe change this sentence to say it should only be called on an MD that's already been validated by `IsValidAndSigned` (implying that the error of that function was checked, rather than that function had side effects).